utils/build-script

```diff
@@ -552,6 +558,11 @@ also build for Apple watchos, but disallow tests that require an watchOS device"
         default="false",  # so CMake can see the inert mode as a false value
         dest="swift_analyze_code_coverage")
 
+    parser.add_argument("--linux",
+        help="""
+also build for Linux""",
```

Comment:
This description is unclear.  If we are on Linux, then we don't need a parameter.  If we are on a different OS, then `--cross-compile-targets` should be sufficient, right?